### PR TITLE
Support parsing `ALTER PLUGGABLE DATABASE` in Oracle

### PR DESCRIPTION
**Pull request by chakkk309**
Ref https://github.com/apache/shardingsphere/issues/6480 & https://github.com/apache/shardingsphere/issues/18389

- Add  `ALTER PLUGGABLE DATABASE`  grammar
- Support parsing  `ALTER PLUGGABLE DATABASE`  in Oracle
- Add test

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 [hunk 1]
```diff
@@ -3214,3 +3214,193 @@ forRefreshClause
...
+    | DEFAULT TABLESPACE tablespaceName
+    | DEFAULT TEMPORARY TABLESPACE (tablespaceName | tablespaceGroupName)
+    | RENAME GLOBAL_NAME TO databaseName (DOT_ domain)+
+    | setTimeZoneClause
+    | databaseFileClauses
+    | supplementalDbLogging
+    | pdbStorageClause
+    | pdbLoggingClauses
+    | pdbRefreshModeClause
+    | REFRESH pdbRefreshSwitchoverClause?
```
Comment (chakkk309) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3261]: According to Oracle's official documentation for [ALTER PLUGGABLE DATABASE](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/ALTER-PLUGGABLE-DATABASE.html#GUID-A29491AD-8F0F-4E52-9D94-57FC3FF8FBC7), it is found that there are some differences between BNF syntax and .eps syntax for **pdb_settings_clauses**.
- in BNF syntax:
524f22a107e9c78b9a3b2d49b8d51d0

- in .eps syntax,  there is an optional clause after the refresh keyword : 

a47ae24adfee4b8ac5e8af4b3452a12
I would like to know which kind of syntax should I follow?

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 [hunk 2]
```diff
@@ -3214,3 +3214,193 @@ forRefreshClause
...
+snapshotClauses
+    : pdbSnapshotClause
+    | materializeClause
+    | createSnapshotClause
+    | dropSnapshotClause
+    | setMaxPdbSnapshotsClause
+    ;
+
+pdbSnapshotClause
+    : ENABLE SNAPSHOT (MANUAL | (EVERY snapshotInterval (HOURS | MINUTES)) | NONE)
```
Comment (chakkk309) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3388]: And in snapshotClauses, there also have differences: 
- in graphic syntax diagrams:
image

- in the description of image, there is an additional `ENABLE` keyword:
38acb62dff94d19d531e2f635fe450a

Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3388]: I think you can execute these sqls in Oracle database.

Comment (chakkk309) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3388]: Thanks, I will check these in Oracle.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 (hunk 1 above)
Comment (chakkk309) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3261]: Update: I found the usage of **pdb_refresh_switchover_clause** in ALTER PLUGGABLE DATABASE in the official documentation, so we should keep this SQL syntax.

Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:3261]: > Update: I found the usage of **pdb_refresh_switchover_clause** in ALTER PLUGGABLE DATABASE in the official documentation, so we should keep this SQL syntax.

That's great.

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Can you solve code conflict first?

**Comment by chakkk309** [comment]
> Can you solve code conflict first?

Sure, solved it.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 (hunk 2 above)
Comment (chakkk309) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4]: According to the SQL execution result, we should remove the `ENABLE` keyword.

5e87aeebbc0b2261b26d42b17526f85

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**